4-projects/README.md

```diff
@@ -94,3 +94,7 @@ When using Cloud Build or Jenkins as your CI/CD tool each environment correspond
 1. Run `./tf-wrapper.sh apply development`
 
 If you received any errors or made any changes to the Terraform config or `terraform.tfvars` you must re-run `./tf-wrapper.sh plan <env>` before run `./tf-wrapper.sh apply <env>`
+
+### Optional Variables
+
+Some variables used to deploy the step have default values, check if you want to change those values before doing the deploy. For more information check in the section **Inputs** the variables not required for the environments [development](./business_unit_1/development/README.md#Inputs), [non-production](./business_unit_1/non-production/README.md#Inputs) and [production](./business_unit_1/production/README.md#Inputs).
```

Comment:
nit: If @atos-cit agrees with my feedback here https://github.com/terraform-google-modules/terraform-example-foundation/pull/239, I think we can remove this.

Reply:
Removed the section as agreed in #239 .